.pre-commit-config.yaml

```diff
@@ -43,5 +43,4 @@ repos:
     hooks:
       - id: black
         name: Format code
-        args: [--skip-string-normalization, --line-length=119]
```

Comment:
Good catch; guess this shows not many people used `pre-commit` since this wasn't hit earlier.